configure.py

```diff
@@ -1118,6 +1123,7 @@ def configure_v8(o):
   o['variables']['v8_random_seed'] = 0  # Use a random seed for hash tables.
   o['variables']['v8_promise_internal_field_count'] = 1 # Add internal field to promises for async hooks.
   o['variables']['v8_use_snapshot'] = 'false' if options.without_snapshot else 'true'
+  o['variables']['v8_use_siphash'] = 'false' if options.without_siphash else 'true'
```

Comment:
For posterity: this could be just `o['variables']['v8_use_siphash'] = b(options.without_siphash)` but it's fine, it's still locally consistent.

Reply:
thanks for the tip, we could do a lot of simplification with that but I'll withhold this time

Reply:
Do we even need a `./configure` flag?
IIUC this is an obscure enough configuration that doing:
```
./configure -- -Dv8_use_siphash=false
```

Reply:
It can be removed in a future version when someone's cleaning up. For now if it ends up causing problems, performance or backward compatibility, or whatever, at least they have an easy out